addons/account/tests/test_account_move_line_tax_details.py

```diff
@@ -901,7 +901,8 @@ def test_fixed_tax_with_negative_balance(self):
                 Command.create({
                     'name': 'line3',
                     'account_id': self.company_data['default_account_revenue'].id,
-                    'price_unit': -400.0,
+                    'price_unit': 400.0,
+                    'quantity': -1,
```

Comment:
@qdp-odoo Because a fix amount "a.k.a recupel" is supposed to scale with the quantity. Here, I changed the test because it doesn't make any sense to have a positive quantity and a negative price_unit. Also, with the current design, the quantity_multiplicator is made before having access to the context for the taxes evaluation

Reply:
you're explaining, yet I don't understand. 

Does it actually make a difference in the computation to have
a) a negative qtty with a positive price_unit
b) a positive qtty with a negative price_unit
? if yes, I say we have a problem. @chklop

Reply:
@smetl @qdp-odoo I'm not sure I follow the discussion here. As the total is always qty*price_unit, whether you have -1 on either side does not make any difference.

Reply:
@chklop The question is about fixed tax. Suppose a recupel of 0.10
qty=1, price_unit=100 => total: 100 + 0.10 = 100.10
qty=1, price_unit=-100 => total: -100 + 0.10 = -90.90
qty=-1, price_unit=100 => total: 100 - 0.10 = 90.90
qty=-1, price_unit=-100 => total: -100 - 0.10 = -100.10